wd_helper.au3

```diff
@@ -750,9 +750,7 @@ EndFunc   ;==>_WD_FrameLeave
 ;                  $bReturnAsArray      - [optional] Return result as array? Default is True.
 ; Return values .: Success - 2D array (with 7 cols) or string ( delimited with | and @CRLF ) @extended contains information about frame count
 ;                  Failure - "" (empty string) and sets @error to one of the following values:
-;                  - $_WD_ERROR_GeneralError
-;                  - $_WD_ERROR_Exception
-;                  - $_WD_ERROR_RetValue
+;                  - $_WD_ERROR_?????????
```

Comment:
need to be rethinked/checked

Reply:
Other functions document this in one of the following manners --

```autoit
; Return values .: Success - <varies by function>
;                  Failure - Response from web driver and sets @error returned from _WD_ExecuteScript()
```

```autoit
; Return values .: Success - Matching Element ID(s)
;                  Failure - @error set to $_WD_ERROR_NoMatch if there are no matches OR
;                            Response from _WD_ExecuteScript() and sets @error to value returned from _WD_ExecuteScript()
```

Reply:
> I hope the final version

Wishful thinking 🤣 

I see you eliminated the code converting the array back into a string. That will lead to incorrect results because you manipulated the array contents following the array creation, correct?

Reply:
Please take a look on: [revert: getting string with recalculated locations](https://github.com/Danp2/au3WebDriver/pull/434/commits/b4d9ce5945fe14a3467546dab173a04b3880a9b1)